test/parallel/test-intl-no-icu-data.js

```diff
@@ -0,0 +1,10 @@
+// Flags: --icu-data-dir=test/fixtures/empty/
```

Comment:
This flag is invalid if Node.js is compiled without intl. and will probably cause the test runner to fail to execute the test (as opposed to gracefully skipping over it).